src/routes/decks/[marketplaceDeckId]/+page.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+  import { user, willAttemptLogin } from "$lib/firebase";
+  import { enhance } from "$app/forms";
+  import type { PageData } from "./$types";
+  import { afterUpdate } from "svelte";
+  import ProBadgeWrapper from "$lib/components/ProBadgeWrapper.svelte";
+  import { goto } from "$app/navigation";
+  export let data: PageData;
+
+  $: {
+    if (!$user && !willAttemptLogin())
+      goto(`/login?successRedirect=${encodeURIComponent(location.pathname)}`);
+  }
+
+  let isPro = false;
+  let isLoading = true;
+
+  afterUpdate(async () => {
+    const idToken = await $user?.getIdToken();
+    const userData = await fetch("/api/user", {
+      headers: { Authorization: `Bearer ${idToken}` },
+    });
+    isPro = (await userData.json()).isSubscribedToBlendPro;
+    isLoading = false;
+  });
+</script>
+
+<svelte:head>
+  <title>Deck Import</title>
+</svelte:head>
+
+<main>
+  <div class="content">
+    {#if isLoading}
+      <h2 style="font-family: 'Contrail One'">Loading...</h2>
+    {:else}
+      <div class="side-by-side">
+        <div class="info">
+          <h1>{data.deckMetadata.name}</h1>
+          <p style="margin-top: 0;">Created by: {data.deckMetadata.author}</p>
+          <p>{data.deckMetadata.description}</p>
+        </div>
+        <img src={data.deckMetadata.image} alt="deck letters" />
+      </div>
+      {#if isPro}
+        <form method="POST" use:enhance>
+          <fieldset>
+            <input type="hidden" name="uid" value={$user?.uid} />
+            <button formaction="?/accept" type="submit" class="btn btn-green"
+              >Import</button
+            >
+          </fieldset>
+        </form>
+      {:else}
+        <ProBadgeWrapper>
+          <!-- TODO: this should point to the new Blend Pro feature page -->
```

Comment:
Not sure if we want to just disable the link entirely or link it out to the Blend Pro Features page that we're adding but i linked it to pricing for now

Reply:
I think we should disable it only because it still says "Import" which is misleading. However we can wrap the `img` tag in the pro wrapper with an `a` tag so make that a link